ord_schema/interface/build_database.py

```diff
@@ -170,20 +171,35 @@ def create_database():
                           port=FLAGS.port)
     cursor = db.cursor()
     if FLAGS.overwrite:
+        logging.info('Removing existing tables')
         for table in _TABLES:
-            cursor.execute(f'DROP TABLE IF EXISTS {table};')
-    cursor.execute('CREATE EXTENSION IF NOT EXISTS rdkit;')
-    cursor.execute('CREATE SCHEMA rdk;')
+            command = sql.SQL('DROP TABLE IF EXISTS {}')
+            cursor.execute(command.format(sql.Identifier(table)))
+    cursor.execute(sql.SQL('CREATE EXTENSION IF NOT EXISTS rdkit'))
+    cursor.execute(sql.SQL('CREATE SCHEMA rdk'))
```

Comment:
Minor point, but perhaps the schema name rdk should be defined by an internal constant `_SCHEMA` in this file and not hard-coded here and in subsequent queries, e.g., ` 'rdk.{table}'`

Reply:
Good idea; I'll push a fix.